main_test.py

```diff
@@ -18,54 +18,123 @@
 # These tests are unit tests that mock Pub/Sub.
 
 import base64
-from uuid import uuid4
+import re
 
 import pytest
 
 import main
+import philips_hue
+import philips_hue_mock
 
 
 @pytest.fixture
-def client():
+def config():
+    main.app.config.from_object('config.DevConfig')
```

Comment:
What if we create a TestConfig, with hardcoded constants? So then the config loading codepaths are still exercised, but running the tests are dependent on multiple environment variables being configured just right?

Reply:
Done